website/content/api-docs/system/step-down.mdx

```diff
@@ -12,8 +12,9 @@ The `/sys/step-down` endpoint causes the node to give up active status.
 
 ## Step down leader
 
-This endpoint forces the node to give up active status. If the node does not
-have active status, this endpoint does nothing. Note that the node will sleep
+This endpoint forces the node to give up active status. When executed against a 
+non-active node, i.e. a standby or performance standby node, the request will be 
+forwarded to the active node. Note that the node will sleep
```

Comment:
```suggestion
The step-down endpoint forces nodes to give up "active" status. If you
call the endpoint against a non-active node, for example, a DR or
 performance standby node, Vault forwards the request to the active
  node.
  
    Note that the node will sleep
```
Style correction: write in the active voice, make it clear who/what takes the action